python/cuda_cccl/benchmarks/parallel/bench_reduce.py

```diff
@@ -7,8 +7,8 @@
 
 def reduce_pointer(input_array, build_only):
     size = len(input_array)
-    res = cp.empty(tuple(), dtype=input_array.dtype)
-    h_init = np.zeros(tuple(), dtype=input_array.dtype)
+    res = cp.empty(1, dtype=input_array.dtype)
```

Comment:
Just to be clear, `np.empty(tuple())` creates 0d array, and `np.empty(1)` creates 1d array with a single element. 

It is a matter of taste, of course, which one to use because both represent a single item, but I was using `tuple()` were the result is a scalar.